integration-tests/oidc-client-reactive/pom.xml

```diff
@@ -156,6 +156,9 @@
                         <gradleEnterprise>
                             <normalization combine.children="append">
                                 <runtimeClassPath>
+                                    <ignoredFiles>
+                                        <ignoredFile>META-INF/ide-deps/**</ignoredFile>
+                                    </ignoredFiles>
```

Comment:
@jprinet I couldn't find a combination of `combine.children` that would work here. Problem is that we end up with multiple elements of the same type in the resulting output.

I'm not completely sure if it's something you support?

For now I copied the rule here but interested in your feedback.

Reply:
I applied the `combine.children=append` to the `runtimeClassPath` section instead of the `normalization` section [here](https://github.com/quarkusio/quarkus/pull/37209/files#diff-427209223f0e5a1cbe626296bd8c8fc20adb0ec7b6a244e9a6a1ae7262b15db9R157) which helped me to get all the keys inherited without duplication (when looking at Intellij's effective pom).

See the result:
![Screenshot 2023-11-21 at 9 27 12 AM](https://github.com/quarkusio/quarkus/assets/10243934/5622047c-057d-4d17-8c71-396283c0630d)


Is this what you were looking for @gsmet ?